protocols/luci-proto-yggdrasil/htdocs/luci-static/resources/protocol/yggdrasil.js

```diff
@@ -0,0 +1,272 @@
+				cell.className = "td"
+				cell.textContent = '%t'.format(peer.uptime);
+
+				cell = row.insertCell(-1)
+				cell.className = "td"
+				cell.textContent = '%.2mB'.format(peer.bytes_recvd);
+
+				cell = row.insertCell(-1)
+				cell.className = "td"
+				cell.textContent = '%.2mB'.format(peer.bytes_sent);
+
+				cell = row.insertCell(-1)
+				cell.className = "td"
+				cell.textContent = peer.priority;
+
+				cell = row.insertCell(-1)
+				cell.className = "td"
+				if (!peer.up) {
+					cell.innerHTML = "<u style='cursor: default'>%t ago</u>".format(peer.last_error_time)
+					cell.dataToggle = "tooltip"
+					cell.title = peer.last_error
+				} else {
+					cell.innerHTML = "-"
+				}
+			});
+			setTimeout(updateActivePeers.bind(this, ifname), 5000);
+		}
+	});
+}
+
+
+var cbiActivePeers = form.DummyValue.extend({
+	cfgvalue: function(section_id, value) {
+		updateActivePeers(this.option);
+		return E('table', {
+			'class': 'table',
+			'id': 'yggdrasil-active-peerings-' + this.option,
+		},[
+			E('tr', {'class': 'tr'}, [
+				E('th', {'class': 'th'}, _('URI')),
+				E('th', {'class': 'th'}, _('State')),
+				E('th', {'class': 'th'}, _('Dir')),
+				E('th', {'class': 'th'}, _('IP Address')),
+				E('th', {'class': 'th'}, _('Uptime')),
+				E('th', {'class': 'th'}, _('RX')),
+				E('th', {'class': 'th'}, _('TX')),
+				E('th', {'class': 'th'}, _('Priority')),
+				E('th', {'class': 'th'}, _('Last Error')),
+			])
+		]);
+	}
+});
+
+var generateKey = rpc.declare({
+	object:'luci.yggdrasil',
+	method:'generateKeyPair',
+	expect:{keys:{}}
+});
+
+var getPeers = rpc.declare({
+	object:'luci.yggdrasil',
+	method:'getPeers',
+	params:['interface'],
+	expect:{peers:[]}
+});
+
+return network.registerProtocol('yggdrasil',
+	{
+		getI18n: function() {
+			return _('Yggdrasil Network');
+		},
+		getIfname: function() {
+			return this._ubus('l3_device') || this.sid;
+		},
+		getType: function() {
+			return "tunnel";
+		},
+		getOpkgPackage: function() {
+			return 'yggdrasil';
+		},
+		isFloating: function() {
+			return true;
+		},
+		isVirtual: function() {
+			return true;
+		},
+		getDevices: function() {
+			return null;
+		},
+		containsDevice: function(ifname) {
+			return(network.getIfnameOf(ifname)==this.getIfname());
+		},
+		renderFormOptions: function(s) {
+			var o, ss;
+			o=s.taboption('general',form.Value,'private_key',_('Private key'),_('The private key for your Yggdrasil node'));
+			o.optional=false;
+			o.password=true;
+			o.validate=validatePrivateKey;
+
+			o=s.taboption('general',form.Value,'public_key',_('Public key'),_('The public key for your Yggdrasil node'));
+			o.optional=false;
+			o.validate=validatePublicKey;
+
+			s.taboption('general',cbiKeyPairGenerate,'_gen_server_keypair',' ');
+
+			o=s.taboption('advanced',form.Value,'mtu',_('MTU'),_('Specify an MTU (Maximum Transmission Unit) for your local TUN interface. Default is the largest supported size for your platform. The lowest possible value is 1280.'));
```

Comment:
Can MTU explanation be shortened? Does ygg's MTU meaning differ to how MTU is described elsewhere on the platform?

Reply:
Yes by default (and as a feature) the MTU is maxed to `65535`

- [ ] We should reduce the text to quickly explain the default 65535 is optimal for yggdrasil

Reply:
changed placeholder to `65535`
added `A default MTU of 65535 is set by Yggdrasil. It is recomended to utilize the default.`